docs/source/configuration.md

```diff
@@ -6,17 +6,17 @@ Following fields are required (and can be customized) in the [`challenge_config.
 
 - **short_description**: Short description of the challenge (preferably 140 characters max)
 
-- **description**: Long description of the challenge (use a relative path for the HTML file, e.g. `challenge_details/description.html`)
+- **description**: Long description of the challenge (use a relative path for the HTML file, e.g. `templates/description.html`)
 
-- **evaluation_criteria**: Evaluation criteria and details of the challenge (use a relative path for the HTML file, e.g. `challenge_details/evaluation.html`)
+- **evaluation_details**: Evaluation details and details of the challenge (use a relative path for the HTML file, e.g. `templates/evaluation_details.html`)
 
-- **terms_and_conditions**: Terms and conditions of the challenge (use a relative path for the HTML file, e.g. `challenge_details/tnc.html`)
+- **terms_and_conditions**: Terms and conditions of the challenge (use a relative path for the HTML file, e.g. `templates/terms_and_conditions.html`)
 
-- **image**: Logo of the challenge (use a relative path for the logo in the zip configuration, e.g. `images/logo/challenge_logo.jpg`). **Note**: The image must be in jpg, jpeg or png format.
+- **image**: Logo of the challenge (use a relative path for the logo in the zip configuration, e.g. `./logo.jpg`). **Note**: The image must be in jpg, jpeg or png format.
```

Comment:
Why was the path here changed?

Reply:
The  path change was due to the location of image in the githiv template, but seeing this now it was only an example of relative path, I will change the path back to original in all locations.